### PR TITLE
Subpart CSS changes

### PR DESCRIPTION
**Pull request by donjo**
Fixes eregs/notice-and-comment#311

screen shot 2016-05-27 at 3 31 13 pm

### CODE REVIEW

File: regulations/static/regulations/css/less/base-variables.less [hunk 1]
```diff
@@ -21,7 +21,7 @@ variables.less contains all theme variable / variable overrides
 @blue: #0072CE;
 @link_blue: #348dc9; /* blue used for link borders and search results */
 @light_neutral: #D7D2CB;
-@dark_gray: #797D81
+@dark_gray: #797D81;
```
Comment (cmc333333) [inline regulations/static/regulations/css/less/base-variables.less]: Sharp eyes!

File: regulations/static/regulations/css/less/module/comment.less [hunk 2]
```diff
@@ -94,20 +94,29 @@ Preamble Read Mode
   .section-title,
   .subpart-addition,
   .subpart-info {
-    width: 75%;
+    width: 100%;
   }
```
Comment (cmc333333) [inline regulations/static/regulations/css/less/module/comment.less]: I think we can remove this rule (and 75% rule for `cfr-instructions`) altogether; div's expand to full width without explicitly being told to. Now that everything is contained within the 75% div, we should be good to go.

**Comment by donjo** [comment]
oh yeah, good catch. should be ready now.

**Comment by cmc333333** [comment]
Rockin

**Pull request merged**